com.jetbrains.RustRover.yaml

```diff
@@ -12,21 +12,24 @@ finish-args:
   - --share=network
   - --socket=pulseaudio
   - --filesystem=host
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
Why? This is not allowed by default.

Reply:
I'm new to building Flatpak app. Actually, I'm not sure if org.freedesktop.Flatpak is required, but it's mentioned in many other JetBrains Flatpak apps, except for PyCharm and GoLand. I removed it in https://github.com/flathub/flathub/pull/4597/commits/96ae5278dedd885c64e6ed36bb0565179604a131 commit.